test/onnx/onnx_import_reshape.in.cpp

```diff
@@ -424,20 +424,6 @@ NGRAPH_TEST(${BACKEND_NAME}, onnx_model_concat_negative_axis)
     test_case.run();
 }
 
-NGRAPH_TEST(${BACKEND_NAME}, onnx_model_flatten)
```

Comment:
Why should we remove a static test for flatten?

Reply:
as above